src/main/java/org/checkerframework/specimin/UnsolvedClassOrInterface.java

```diff
@@ -60,7 +63,7 @@ public UnsolvedClass(String className, String packageName) {
    * @param packageName the name of the package
    * @param isException does the class represents an exception?
    */
-  public UnsolvedClass(String className, String packageName, boolean isException) {
+  public UnsolvedClassOrInterface(String className, String packageName, boolean isException) {
```

Comment:
Rather than duplicating the code in this constructor in the new one, this constructor should be a `this(...)` call, just like the constructor directly above it, that adds `false` to its parameter list.